src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIPacket.cs

```diff
@@ -183,21 +157,11 @@ public void AppendData(ReadOnlySpan<byte> data)
         }
```

Comment:
This method `public void AppendData(ReadOnlySpan<byte> data)`  is unused now. This can be deleted.

Reply:
Done.